propertyestimator/layers/simulation.py

```diff
@@ -683,7 +683,7 @@ def schedule_calculation(calculation_backend, storage_backend, layer_directory,
         force_field_path = path.join(layer_directory, 'force_field_{}'.format(data_model.parameter_set_id))
 
         with open(force_field_path, 'wb') as file:
-            pickle.dump(force_field, file)
+            pickle.dump(serialize_force_field(force_field), file)
```

Comment:
I'm not highly experienced with serialization, but this seems like it might be an unnecessary use of pickle. Is there anything wrong with `file.write(serialize_force_field(force_field))`? Pickle potentially introduces python-only behavior (https://www.datacamp.com/community/tutorials/pickle-python-tutorial#not)

Also, I generally try to avoid using `file` as a variable because Python seems to have it as a reserved keyword (I've never used the keyword, but the syntax highlighting indicates a potential problem). Maybe consider `file_obj` or `fo`.

Reply:
I think the main reason is that the storage layer is written as an abstraction, which while the currently employed implementation stores objects on the disk, future implementations may store objects with a database for example. Using pickle gives this flexibility for moving away from local files 'easily'! But there could be a better way to handle this?

Reply:
Also I thought file was a builtin in Python 2 which got removed in 3? Could be wrong about this though!

Reply:
Ah, so `file` is a holdover from Python 2. I'm learning new things!